client/components/tinymce/plugins/mentions/mentions.jsx

```diff
@@ -82,9 +74,9 @@ const Mentions = React.createClass( {
 			showPopover: typeof query === 'string',
 			query,
 		} );
-	},
+	}
```

Comment:
Would that work, though? Doesn't calling `onKeyUp` produce new function each time? If that's the case, for performance reasons, it's better to bind the function once in the constructor than to re-create the function on each call.